app/src/main/java/org/listenbrainz/android/viewmodel/SearchViewModel.kt

```diff
@@ -184,4 +190,9 @@ class SearchViewModel @Inject constructor(
             inputQueryFlow.emit("")
         }
     }
+    
+    /** True if the error states the the user is already being followed.*/
+    private fun userIsAlreadyFollowed(error: ResponseError?): Boolean =
+        error == ResponseError.BAD_REQUEST &&
+                error.actualResponse?.contains("already following") == true
```

Comment:
avoid this as said before

Reply:
This is a temporary solution until we the api itself is updated or we have profile screen in the app so we can remove the follow button all together.